src/wp-includes/class-wp-scripts.php

```diff
@@ -133,6 +133,24 @@ class WP_Scripts extends WP_Dependencies {
 	 */
 	private $type_attr = '';
 
+	/**
+	 * Holds a mapping of dependents (as handles) for a given script handle.
+	 * Used to optimize recursive dependency tree checks.
+	 *
+	 * @since 6.3.0
+	 * @var array
+	 */
+	private $dependents_map = array();
+
+	/**
+	 * Holds a reference to the allowed script loading strategies.
+	 * Used by methods that validate loading strategies.
+	 *
+	 * @since 6.3.0
+	 * @var array
```

Comment:
```suggestion
	 * @var string[]
```

Reply:
@westonruter addressed in https://github.com/WordPress/wordpress-develop/commit/d17203e911fccaed4d584feb4e0f8759f3fae590